src/explore-education-statistics-admin/src/pages/admin-dashboard/AdminDashboardPage.tsx

```diff
@@ -112,7 +118,7 @@ const AdminDashboardPage = () => {
               id="my-publications"
               title="Manage publications and releases"
             >
-              <MyPublicationsTab />
+              <MyPublicationsTab themePropId={themeId} topicPropId={topicId} />
```

Comment:
Probably best not to call them "props" explicitly, as we don't do this anywhere else - i.e. instead of "themePropId", just call it "themeId"